workflows/repeatmasking/Repeat-masking-with-RepeatModeler-and-RepeatMasker.ga

```diff
@@ -0,0 +1 @@
axy/mutable-config/tool-data/shared/ucsc/chrom/?.len\", \"input_file\": null, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "2.0.4+galaxy1", "type": "tool", "uuid": "9312ba36-4275-4d40-8ba6-95eea1b23b11", "when": null, "workflow_outputs": [{"label": "RepeatModeler seeds alignments", "output_name": "seeds", "uuid": "b5cd1bca-27c7-4887-904d-f59084d68291"}, {"label": "RepeatModeler consensus sequences", "output_name": "sequences", "uuid": "65e194d1-db7c-4783-97c4-d5051e4b64ea"}]}, "2": {"annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/4.1.5+galaxy0", "errors": null, "id": 2, "input_connections": {"input_fasta": {"id": 1, "output_name": "sequences"}}, "inputs": [], "label": null, "name": "RepeatMasker", "outputs": [{"name": "output_masked_genome", "type": "fasta"}, {"name": "output_log", "type": "tabular"}, {"name": "output_table", "type": "txt"}, {"name": "output_repeat_catalog", "type": "txt"}, {"name": "output_gff", "type": "gff"}], "position": {"left": 440, "top": 0}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/4.1.5+galaxy0", "tool_shed_repository": {"changeset_revision": "ba6d2c32f797", "name": "repeat_masker", "owner": "bgruening", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"__input_ext\": \"input\", \"advanced\": {\"is_only\": false, \"is_clip\": false, \"no_is\": false, \"rodspec\": false, \"primspec\": false, \"nolow\": false, \"noint\": false, \"norna\": false, \"alu\": false, \"div\": false, \"search_speed\": \"\", \"frag\": \"40000\", \"gc\": null, \"gccalc\": false, \"nocut\": false, \"xout\": false, \"keep_alignments\": false, \"invert_alignments\": false, \"poly\": false}, \"chromInfo\": \"/shared/ifbstor1/galaxy/mutable-config/tool-data/shared/ucsc/chrom/?.len\", \"excln\": true, \"gff\": true, \"input_fasta\": null, \"repeat_source\": {\"source_type\": \"dfam\", \"__current_case__\": 0, \"species_source\": {\"species_from_list\": \"no\", \"__current_case__\": 1, \"species_name\": \"\"}}, \"xsmall\": true, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "4.1.5+galaxy0", "type": "tool", "uuid": "e6c8e6a1-efe8-4291-b12b-5fdb3795b6ca", "when": null, "workflow_outputs": [{"label": "RepeatMasker output log", "output_name": "output_log", "uuid": "8b69884b-5990-48f6-aa80-cf705586b313"}, {"label": "RepeatMasker repeat annotation", "output_name": "output_gff", "uuid": "6b96cc61-19be-4abd-b99d-7639ef6ed51f"}, {"label": "RepeatMasker masked genome", "output_name": "output_masked_genome", "uuid": "b0e0f699-3085-424d-82f0-b9c57361e8bb"}, {"label": "RepeatMasker repeat catalog", "output_name": "output_repeat_catalog", "uuid": "02b790ca-b3e7-441c-baf0-f3c1942cb8ca"}, {"label": "RepeatMasker repeat statistics", "output_name": "output_table", "uuid": "eb04a2c3-2461-4f8f-9f3f-c42143f79904"}]}}, "tags": [], "uuid": "f25be8fa-7823-456f-9707-a497703f48d7", "version": 0}
```

Comment:
@rlibouba Is this file really used? There's another .ga file in your PR

Reply:
I deleted it, I hadn't seen it